models/_sector/cex/flows/cex_celo_flows.sql

```diff
@@ -0,0 +1,18 @@
+{% set blockchain = 'celo' %}
+
+{{ config(
+        
+        schema = 'cex_' + blockchain,
+        alias = 'flows',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['flow_type', 'unique_key']
+)
+}}
+
+{{cex_flows(
+        blockchain = blockchain
+        , transfers = ref('tokens_scroll_transfers')
+        , addresses = ref('cex_scroll_addresses')
```

Comment:
still scroll here

Reply:
changed to this in all `cex_{chain}.flows` spells for easier management:
```
, transfers = ref('tokens_' + blockchain + '_transfers')
, addresses = ref('cex_' + blockchain + '_addresses')
```

Reply:
love it!